librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -170,6 +170,44 @@ commands:
           - RZ_OUTPUT_MODE_STANDARD
           - RZ_OUTPUT_MODE_RIZIN
         args: []
+  - name: ad
+    summary: Analyze data
+    subcommands:
+      - name: ad
+        summary: Analyze [count] data words with [depth]
+        cname: analysis_data
+        args:
+          - name: count
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+          - name: depth
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+          - name: wordsize
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: adf
+        summary: Analyze data in function
+        cname: analysis_data_function
+        args: []
+      - name: adfg
+        summary: Analyze data in function gaps
+        cname: analysis_data_function_gaps
+        args: []
+      - name: adk
+        summary: Analyze data kind (code, text, data, invalid, etc)
+        cname: analysis_data_kind
+        args: []
+      - name: adt
+        summary: Analyze data trampolines
```

Comment:
Do you know enough about this cmd to expand a bit this description? It's not clear what it does with the arguments.

Reply:
I have zero clue how it works, there are no tests, but I didn't remove it completely since from the code, it *might* be useful.